lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -307,6 +360,24 @@ def set_fn(value):
             self._fig = plt.figure()
             self._axes = self._fig.add_subplot(111)
 
+    def _select_state(self, value):
+        """Select a different state to operate the widget in. Note that the input argument is value
```

Comment:
```suggestion
        """Select a different state to operate the widget in. Note that the input argument is `value`
```
Alternatively, perhaps rename `value` to `input_argument` to obsolete the "Note" comment altogether.

Reply:
Unfortunately, the callback from `ipywidgets.RadioButtons.observe()` always calls with `value`. I'm not sure whether this is alterable.